addons/account/models/company.py

```diff
@@ -248,8 +248,13 @@ class ResCompany(models.Model):
     account_discount_expense_allocation_id = fields.Many2one(comodel_name='account.account', string='Separate account for expense discount')
 
     # Audit trail
-    check_account_audit_trail = fields.Boolean(string='Audit Trail')
-
+    forced_gobd_compliant = fields.Boolean(
+        string='Forced GoBD Compliant',
```

Comment:
Also FYI when you create field, if you hesitate between using double quotes or single quotes there s a non formal convention where we put double quotes for element that the user will see like string, help, etc and single quotes otherwise :smile: (not mandatory to follow ofc)